app/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from . views import get_aliases, get_domains, get_alias_filtered
+urlpatterns = [
+    path('aliases', get_aliases, name="get_aliases"),
+    path('aliases/domains', get_domains, name="get_domains"),
```

Comment:
`path('aliases/domains', get_domains, name="get_domains"),`
should be
`path('domains', get_domains, name="get_domains"),`

Reply:
We don't really require this endpoint either